state-chain/custom-rpc/src/lib.rs

```diff
@@ -209,6 +209,13 @@ pub struct RpcLiquidityProviderBoostPoolInfo {
 	pub is_withdrawing: bool,
 }
 
+#[derive(Serialize, Deserialize, Clone)]
+pub struct RpcAffiliateDetails {
+	pub short_id: AffiliateShortId,
+	pub withdrawal_address: EthereumAddress,
+	pub earned_fees: NumberOrHex,
```

Comment:
```suggestion
	pub balance: NumberOrHex,
```

Feedback was that `earned_fees` was misleading because it didn't include 'all time' fees.

Reply:
I think balance is misleading since it is not the on chain balance but the collected fees in usdc